source/SongCore/UI/RequirementsUI.cs

```diff
@@ -81,6 +86,7 @@ internal void Setup()
             GetIcons();
             standardLevel = Resources.FindObjectsOfTypeAll<StandardLevelDetailViewController>().First();
             tweenyManager = Object.FindObjectOfType<TweeningManager>();
+            loader = Resources.FindObjectsOfTypeAll<CustomLevelLoader>().FirstOrDefault();
```

Comment:
Should probably avoid using `Resources.FindObjectsOfTypeAll<T>.FirstOrDefault()` whenever possible, since it would go through all disabled objects as well, which is extremely slow. If you know it is active, you should use `Object.FindObjectOfType<T>` instead. And you should actually probably just make `Loader._customLevelLoader` `internal` in your specific case, so you avoid the call entirely.